include/apple2.h

```diff
@@ -95,7 +151,12 @@
 #define CH_LTEE         '+'
 #define CH_RTEE         '+'
 #define CH_CROSS        '+'
-#endif
+#define _chline(length)         dyn_chline(CH_HLINE|0x80, length)
+#define _chlinexy(x, y, length) dyn_chlinexy(CH_HLINE|0x80, x, y, length)
+#define _cvline(length)         dyn_cvline(CH_VLINE|0x80, length)
+#define _cvlinexy(x, y, length) dyn_cvlinexy(CH_VLINE|0x80, x, y, length)
```

Comment:
I can't follow you with the two sets of _c[h|v]line<xy> macros. No matter if DYN_BOX_DRAW is defined or not, the user is suppoed to be able to write i.e. cputc(CH_HLINE). He's not supposed to have to write cputc(CH_HLINE_SCRCODE) or cputc(CH_HLINE|0x80). In the end, _c[h|v]line<xy>() call internally something like cputc(). So I don't see why what works for cputc() doesn't also work as-is for _c[h|v]line<xy>().

Reply:
I see. CH_HLINE isn't a constant, so the compiler has to generate code for the things you write in the macros - for each and every call site! Wouldn't it be much better to rather have a
`eor     #$80            ; Invert high bit`
inside the dyn_...() functions?